modules/apps/knowledge-base/knowledge-base-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Knowledge Base API
 Bundle-SymbolicName: com.liferay.knowledge.base.api
-Bundle-Version: 22.0.1
+Bundle-Version: 23.0.0
```

Comment:
The code LGTM

Reply:
You are right. However, I introduced this new `updateStatus` method in the recent commit for the KB Recycle Bin, so I was thinking the Breaking Change notice is not needed, as it will not be published in any release.

Reply:
I do not know about it. Sorry
you should ask higher rank to verify this :( 

When this matter is resolved, please fell free to forward this PR.

Reply:
Thanks Alicia, I agreed with Adolfo that this time the BR notice is not needed because the previous commit is very recently, but generally we should always do it.